app/src/androidTest/java/com/sdpteam/connectout/chat/ChatActivityTest.java

```diff
@@ -65,11 +65,14 @@ public void testListUsersDisplayed() {
 
         onView(withId(R.id.chat_input)).perform(typeText("Excellent Idea"));
         onView(withId(R.id.chat_fab)).perform(click());
+        SystemClock.sleep(1000);
         checkListViewValue(5, "Excellent Idea", "You");
 
         onView(withId(R.id.chat_input)).perform(typeText("Hi beautiful people"));
         onView(withId(R.id.chat_fab)).perform(click());
+        SystemClock.sleep(1000);
         checkListViewValue(6, "Hi beautiful people", "You");
+        model.emptyTestMode();
```

Comment:
Maybe it's better to put this in the @After, because if the test doesn't finish for some reason, this cleaning function in never call, no?

Reply:
yup I'll change that now